### ISSUE TITLE
ConcurrentModificationException during startup chunk loading

### ISSUE THREAD

**Issue by Abakus2k** [Cazadorsniper/PO3#1695]
**Pack Version: 3.4.9**
**Normal**
**Multiplayer**
(maybe related to https://github.com/Cazadorsniper/PO3/issues/1683)

**Description:**
After running the server for about a week, it crashed after restart.
The crash appears on every startup, right when loading the overworld chunks.
It may have to do something with an issue that occured with AE2 + Myst. Agriculture Auto-Crafting.
We experienced a bug where a "master infusion crystal" from MA was stuck in our AE2 system.
The crystal was "stuck" in the system, with us not being able to remove it.
Putting it with a pattern in AE2s molecular assembler/me interface might have caused this issue.
I was not able to reproduce this in singleplayer.
If there are more logs needed, Im happy to provide them.
All our backups are unusable and it would be a pain losing weeks of progress.
For any ideas on how to fix our savefiles I would be thankful.

**Steps to reproduce:**
???

**Crash log (use pastebin or issue will be closed):**
https://pastebin.com/vRTcK1kv

**Comment by Cazadorsniper** [comment]
You can try loading a backup save.

**Comment by Abakus2k** [comment]
Thank you for your quick reply. The same behaviour is consistent on startup with all our backups, making all of the useless.
Probably the savefile was corrupted days ago, but it didnt come to light since the server was never restarted.
Ive since also tried loading the world in singleplayer with AE2 and/or MA disabled. Same error code.
Any further idea how to better debug? The error code is kinda undescriptive.
I can also provide the world-files if needed.

**Comment by Cazadorsniper** [comment]
I remember someone having this error before but it had to do with armor+ not myst and AE2.

**Comment by Cazadorsniper** [comment]
The solution for that was to load the world without the mod then put it back then reload the world again.

**Comment by Abakus2k** [comment]
After a little break, we decided to tackle it again. After countless tries and many mods removed each time, we found Mekanism to be the issue. Disabling mekanism, starting the server and then enabling it again fixed the issue. Seems like the problem was Mekanism. Might be just the side-effect of some other mod that uses power clashing with mekanisms power cables, but we weren't able to determine that.
